themes/default/content/blog/project-scoped-stacks-in-self-managed-backend/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Projects in Self-Managed State Backends"
+date: 2023-03-29
+meta_desc: Pulumi's Self-Managed backend now supports stack names scoped by project name.
+meta_image: meta.png
+authors:
+    - justin-vanpatten
+    - abhinav-gupta
+tags:
+    - features
+    - pulumi-releases
+---
+
+At Pulumi, we aim to offer our users the best-in-class IaaC experience.
+We recently shipped a series of impactful enhancements to our Service backend
+(check out [Pulumi Deployments](/blog/pulumi-deployments/) if you haven't already!),
+and we have more exciting features coming soon.
+
+While our Service backend remains the best place to store your Pulumi state,
+we also offer a Self-Managed backend for those who prefer to do it themselves.
+We're committed to giving you the best experience regardless of which backend you use,
+so we've been hard at work improving the Self-Managed backend as well.
+
+Therefore, we're very excited to announce that with the latest release of Pulumi,
+we have added support for **Project-Scoped Stacks** in our Self-Managed backend.
+
+<!--more-->
+
+## Background
+
+This section goes over some of the background necessary
+to understand project-scoping and why it's important.
+Feel free to skip ahead to the next section if you're already familiar with this.
+
+### Projects and stacks
+
+Pulumi [*projects*](/docs/intro/concepts/project/) are a unit of organization.
+A project holds the code and configuration for your infrastructure,
+and can be used to deploy multiple stacks.
+A [*stack*](/docs/intro/concepts/stack/) is a single,
+isolated instance of your infrastructure.
+
+For example, you might have an "app" project for your application,
+deploying to the "test" and "prod" stacks.
+
+![Projects and stacks](projects-and-stacks.png)
+
+### What's a state backend?
+
+Each stack tracks metadata about your infrastructure
+so it can manage your cloud resources.
+This metadata is called [*state*](/docs/intro/concepts/state/)
+and it is stored in a *backend* of your choosing:
+**Service** or **Self-Managed**.
+
+The **Service** backend is hosted at [`app.pulumi.com`](https://app.pulumi.com)
+or [on your own server](https://www.pulumi.com/product/self-hosted/).
+It provides the best combination of usability, safety, and security for most users.
+
+The **Self-Managed** backend stores state in a JSON file,
+deferring to you to manage persistence of this information.
+You can store it in AWS S3, Azure Blob Store, Google Cloud Storage,
```

Comment:
FWIW, the actual name of the thing is "Amazon S3". I almost always have to look these things up for myself, as they [tend to follow a pattern](https://stackoverflow.com/questions/33125790/why-some-services-are-called-aws-xxx-and-the-others-amazon-xxx) ... but of course not always. 😄 https://aws.amazon.com/documentation-overview/

```suggestion
You can store it in Amazon S3, Azure Blob Store, Google Cloud Storage,
```

Reply:
Oops, yeah, let me fix this. S3 and VPC are both "Amazon", not "AWS".